frontend/pulse3d/src/serviceWorker.js

```diff
@@ -248,7 +248,7 @@ self.onmessage = ({ data, source }) => {
     });
   } else if (data.msgType === "stayAlive") {
     // TODO should have this do something else so that there isn't a log msg produced every 20 seconds
-    console.log("Staying alive");
+    // console.log("Staying alive");
```

Comment:
can you add this back in?

Reply:
When you do remove things like this, you should also remember to remove the comments associated with them. This is mentioned in one of the comments above as well

Reply:
I did this one, I thought only part of the branch would be used. My bad